Dockerfile

```diff
@@ -5,15 +5,18 @@ ENV GO111MODULE=on
 ENV CGO_ENABLED=0
 
 # Build templater
-WORKDIR "/go/src/github.com/hashicorp/crt-core-helloworld"
+WORKDIR "/go/src/github.com/hashicorp/actions-packaging-linux"
 COPY ./ .
 RUN go build -o nfpm_template
 RUN cp nfpm_template /usr/local/bin/nfpm_template
 RUN chmod +x /usr/local/bin/nfpm_template
 
 # Download nfpm
-RUN curl -Lo nfpm.tar.gz https://github.com/goreleaser/nfpm/releases/download/v2.13.0/nfpm_2.13.0_Linux_x86_64.tar.gz \
-    && tar -xf nfpm.tar.gz \
+# RUN curl -Lo nfpm.tar.gz https://github.com/goreleaser/nfpm/releases/download/v2.13.0/nfpm_2.13.0_Linux_x86_64.tar.gz \
+#     && tar -xf nfpm.tar.gz \
+RUN git clone --depth=1 --branch fix_deb_version_control https://github.com/kpenfound/nfpm.git \
```

Comment:
Is there a reason we're running this from your acct rather than the main fork or a hashicorp fork?